doc/engineering-reference/src/simulation-models-encyclopedic-reference-002/heat-exchangers.tex

```diff
@@ -953,6 +957,11 @@ \subsubsection{Long Time-Step Response Factors}\label{long-time-step-response-fa
 
 \(d(u_i, u_j^{'})\) is the distance between current point and point on other imaginary borehole.
 
+For the UBHWTcalc model, EnergyPlus builds on the model developed by Cimmino (2018a, 2018b, 2019), with improvements (Cook and Spitler 2021) to reduce memory consumption and increase speed. The general approach is similar to that of the UHFcalc model, but the heat flux is adjusted for each segment to enforce the uniform borehole wall temperature model. Additionally, the methodology of Cimmino (UBHWTcalc) is a semi-analytical solution, so the solution is dependent on the number of segments used. The number of segments are adaptively discretized to approximate a uniform inlet fluid temperature (UIFT) \emph{g}-function. 
```

Comment:
Describes UBHWTcalc as superior to UHFcalc and discusses UBHWTcalc's approximation of UIFT.